WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadUtils.java

```diff
@@ -86,6 +86,37 @@ UiString getErrorMessageResIdFromPostError(boolean isPage, PostError error) {
             case GENERIC_ERROR:
             default:
                 AppLog.w(T.MAIN, "Error message: " + error.message + " ,Error Type: " + error.type);
+                if (eligibleForAutoUpload) {
```

Comment:
This method is used even in failed upload notification, so I've changed even that. But I think it's actually a bonus.